iodata/test/test_mwfn.py

```diff
@@ -0,0 +1,196 @@
+# IODATA is an input and output module for quantum chemistry.
+# Copyright (C) 2011-2019 The IODATA Development Team
+#
+# This file is part of IODATA.
+#
+# IODATA is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# IODATA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+# --
+"""Test iodata.formats.mwfn module."""
+
+
+import numpy as np
+from numpy.testing import assert_equal, assert_allclose
+from ..api import load_one
+from ..formats.mwfn import _load_mwfn_low
+from ..overlap import compute_overlap
+from ..utils import LineIterator
+
+try:
+    from importlib_resources import path
+except ImportError:
+    from importlib.resources import path
+
+
+def helper_load_mwfn_low(fn_mwfn):
+    """Load a testing Gaussian log file with iodata.formats.mwfn.load_mwfn_low."""
```

Comment:
The docstring may need to be fixed. The "Gaussian" part is probably inspired by another format.

Reply:
This function appeared but is not used in the testing anywhere. Possible in preparation for a future use? @FarnazH

Reply:
If I am not wrong, this function was added by you @BradenDKelly, so probably it's leftover from earlier code? I agree with you, it is not used, so it's better to be removed. Thanks.